.gitlab-ci-darwin.yml

```diff
@@ -0,0 +1,95 @@
+variables:
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  GIT_SUBMODULE_STRATEGY: recursive
+  MODULE_CMAKE: "cmake/3.12.4"
+  MODULE_CLANG: "clang/8.0.1"
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
```

Comment:
why `cuda/10.1` and not `cuda/10.2`?

Reply:
@jlippuner had issues with cuda/10.2 https://github.com/lanl/parthenon/pull/114.

Reply:
Ah, it was related to kokkos support for cuda.

Reply:
https://github.com/lanl/parthenon/issues/327

Reply:
Oh I see. Okay. We can update later then, when we update kokkos.